packages/framer-motion/src/events/use-dom-event.ts

```diff
@@ -1,15 +1,5 @@
-import { useEffect, RefObject } from "react"
-
-export function addDomEvent(
```

Comment:
I split this out into its own file as we no longer use `useDomEvent` but we do export it via `index.js` and use it within Framer. This allows us to code split the two functions.